src/applications/static-pages/cta-widget/components/messages/NoMHVAccount.jsx

```diff
@@ -34,6 +48,10 @@ const NoMHVAccount = ({
   return <CallToActionAlert {...content} />;
 };
 
+NoMHVAccount.defaultProps = {
```

Comment:
`defaultProps` were [deprecated from React](https://react.dev/blog/2024/04/25/react-19-upgrade-guide#removed-proptypes-and-defaultprops). You should use ES6 default parameters